projects/bp-gallery/src/components/common/TagOverview.tsx

```diff
@@ -14,6 +14,8 @@ import { useSimplifiedQueryResponseData } from '../../graphql/queryUtils';
 import useGetTagsWithThumbnail from '../../hooks/get-tags-with-thumbnail.hook';
 import PrimaryButton from './PrimaryButton';
 
+export const MAX_TAGS_PER_ROW = 3;
```

Comment:
please fix the linter warning, this doesn't get imported anywhere at the moment, does it?